docs/publicity/announce.txt

```diff
@@ -11,54 +11,30 @@ Compared to the stable 3.0.x series, this version brings too many
 improvements and even more bug fixes to list all of them here, please
 see the change log
 
-https://raw.githubusercontent.com/wxWidgets/wxWidgets/v3.2.0/docs/changes.txt
+https://raw.githubusercontent.com/wxWidgets/wxWidgets/v3.3.0/docs/changes.txt
 
-for the incomplete list of the most important ones and here is the
-maximally condensed summary of the changes compared to 3.0:
+for the list of the most important ones and here is the maximally condensed
+summary of the major changes compared to 3.2:
 
-- Build system improvements: support for new compilers (up to MSVS 2022,
-  g++ 12, clang 14) with an even simpler way of using wxWidgets from
-  MSVS, with wxwidgets.props file, and OS versions as well as an
-  entirely new CMake build system.
-- Support for native dark mode under macOS 10.14 and later, support for
-  ARM hardware and macOS versions up to 13.
-- High DPI support with the new, but almost perfectly backwards-
-  compatible, API based on wxBitmapBundle, including per-monitor DPI and
-  dynamic DPI changes.
-- New features: HTTPS and HTTP/2 support with wxWebRequest and friends;
-  support for freezing rows/columns in wxGrid; mouse gesture events
-  (GSoC 2017 project); non-integer font sizes and arbitrary font weights
-  in wxFont; fractional pen widths in wxGraphicsContext; arbitrary label
-  windows in wxStaticBox; markup in wxDataViewCtrl items text; support
-  for ZIP 64 files; LZMA compression; much improved accessibility
-  support under MSW; new Edge-based wxWebView implementation.
-- New classes: wxActivityIndicator, wxAddRemoveCtrl, wxAppProgressIndicator,
-  wxBitmapBundle, wxNativeWindow, wxPersistentComboBox,
-  wxPowerResourceBlocker, wxSecretStore, wxTempFFile and many new
-  features in the existing classes.
-- New XRC handlers for all the new and some of the existing classes.
-- Significant improvements to: wxBusyInfo, wxDataViewCtrl, wxDirDialog, wxGrid,
-  wxNotificationMessage, wxSpinCtrl, wxStaticBox, wxStyledTextCtrl,
-  wxUIActionSimulator.
-- Improvements to compile-time safety with the possibility to disable dangerous
-  implicit conversions between wxString and "char*" strings.
-- Latest versions of all bundled 3rd party libraries, including all the
-  security fixes and support for WebKit 2 and GStreamer 1.7 under Unix.
-- Revamped OpenGL support better suited to modern OpenGL (3.2+).
-- Further C++11 and later support improvements, wxWidgets can be built using
-  C++20 compilers.
-- New experimental wxQt port.
-- Many, many bug fixes.
+- Support for dark mode in wxMSW under Windows 10 and later.
+- New Chrome-based wxWebView backend and many other wxWebView improvements.
+- Many improvements to wxAUI, including support for pinned and multiline tabs.
+- Support for WebP format in wxImage.
+- Support for high DPI cursors and animations.
+- Latest versions of 3rd party libraries.
+- Support for latest compilers (gcc 15, clang 19/Xcode 16) and C++ standards.
```

Comment:
GCC and clang versions should be bumped up also in the root README:
https://github.com/wxWidgets/wxWidgets/blob/a5bd4eff9e032d7af78f0020647e3b1121d7842c/README.md?plain=1#L39-L40